src/VisualStudio/VisualBasic/Impl/Options/AutomationObject.vb

```diff
@@ -198,16 +198,6 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
             End Set
         End Property
 
-
-        Public Property Style_PreferThrowExpression As String
```

Comment:
I'm not sure what the consequences of removing these properties are. Are they okay to remove or should they be left but marked as obsolete?

Reply:
Good question.  @jasonmalinowski what's the story with these automation objects.  If we remove them, that may break things right?  like with settins sync?

Reply:
What these are used for is importing/exporting from .vssettings files (amongst a few other obscure things.) It doesn't impact setting sync, but removing it would mean a .vssettings file with this contained will not apply the user's setting correctly. I don't know if it'll visibly say that it can't apply or whether it'll just do it silently and leave the user confused.

I also don't know how many people actually *use* .vssettings files...

Reply:
We do still read and write these in the C# `AutomationObject` where they apply though.